base/sort.jl

```diff
@@ -407,6 +408,48 @@ julia> searchsortedlast([1, 2, 4, 5, 5, 7], 0) # no match, insert at start
 ```
 """ searchsortedlast
 
+function insorted(x, coll::AbstractVector, lo::T, hi::T, o::Ordering)::Bool where T<:Integer
```

Comment:
Do we really want this method? AFAICT the corresponding `searchsorted` method is not considered public API.  If a user wants to call a function like this, she would rather invoke `insorted(x, view(coll, lo:hi), ordering=o)`.